### DISCUSSION TITLE
Congratulations to the first epic release!!! Consider another AD system?

### DISCUSSION THREAD

**Discussion by CaiYitao** [LuxDL/Lux.jl#20] [General]
There are plenty of gradient error reports on Zygote.jl which worries people like me, Would you like to consider another much safer AD system?  heard about Enzyme, but not sure if it is better.

**Comment by avik-pal** [comment]
Zygote is the default AD rn only because it integrates well with chainrules and covers all the cases we need.

We will indeed be moving towards Enzyme.jl . FWIW I have tested the latest blas support of enzyme with lux and it seems to work on CPU for simple models.

**Reply by CaiYitao** [reply]
That sounds great! Keep up the great work! ; )  I will try it out in my master's project.

**Reply by CaiYitao** [reply]
Just spotted a typo
<img width="767" alt="image" src="https://user-images.githubusercontent.com/65641330/170357014-6f335ac2-c64b-422e-821a-2e89863677a8.png">

**Reply by avik-pal** [reply]
No that does link to the tutorial from flax
